torchrl/objectives/dqn.py

```diff
@@ -304,13 +301,14 @@ def forward(self, tensordict: TensorDictBase) -> TensorDict:
             pred_val_index = (pred_val * action).sum(-1)
 
         target_value = self.value_estimator.value_estimate(
-            tddevice.clone(False), target_params=self.target_value_network_params
+            td_copy, target_params=self.target_value_network_params
```

Comment:
I think this clone is not needed. 
Pleas double check

Reply:
Also shouldn't this also be wrapped in no_grad()? 
Or the fact that we use target params already takes care of that?

Reply:
yep target are detached